README.md

```diff
@@ -50,6 +50,17 @@ Go through our [file reference](http://kedgeproject.org/file-reference).
 
 Then go further with our pre-existing [examples](https://github.com/kedgeproject/kedge/tree/master/examples).
 
+__Comparison to similar projects:__
+
+|                                          | Kompose | Ksonnet | OpenShift templates | Helm | Kedge |
+|------------------------------------------|---------|---------|---------------------|------|-------|
+| Scalable                                 | -       | ✓       | ✓                   | ✓    | ✓     |
+| Kubernetes compatible API and DSL Syntax | -       | -       | -                   | -    | ✓     |
+| Native to Kubernetes ecosystem           | -       | ✓       | -                   | ✓    | ✓     |
+| Integration to tools (IDEs)              | ✓       | ✓       | ✓                   | ✓    | ✓     |
```

Comment:
I am bit confused here for kompose

Reply:
what @cdrage probably means is that basicaly every IDE that supporsts docker-compose has by definition support for kompose :-)